src/composables/trade/useSor.ts

```diff
@@ -365,11 +376,6 @@ export default function useSor({
         priceImpact.value = 0;
         state.validationErrors.noSwaps = true;
       } else {
-        tokenOutAmount = await adjustedPiAmount(
-          tokenOutAmount,
-          tokenOutAddress
-        );
```

Comment:
I believe the `swapReturn.returnAmount` which is `tokenOutAmount` here used to be the wstETH value that then needed to be converted for the price impact calculation. It now seems that the swapReturn amount is actually correct and so the conversion must be being handled in the SDK.